portal-web/docroot/html/taglib/ui/input_time_zone/page.jsp

```diff
@@ -90,7 +90,7 @@ numberFormat.setMinimumIntegerDigits(2);
 
 			TimeZoneFormat icuTimeZoneFormat = icuSimpleDateFormat.getTimeZoneFormat();
 
-			sb.append(icuTimeZoneFormat.format(TimeZoneFormat.Style.GENERIC_LOCATION, icuTimeZone, date.getTime()));
+			sb.append(icuTimeZoneFormat.format(TimeZoneFormat.Style.ZONE_ID, icuTimeZone, date.getTime()));
```

Comment:
@daledotshan thanks for the explanation.  That makes sense.  I did some re-factoring (un-related), but your changes work well. Thank you!